js/data.js

```diff
@@ -1 +1,48 @@
 /* exported data */
+
+const $teamStandings = document.querySelector('#appendTeams');
+function getPremierLeagueData() {
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://v3.football.api-sports.io/standings?league=39&season=2023');
+  xhr.setRequestHeader('x-rapidapi-host', 'v3.football.api-sports.io');
+  xhr.setRequestHeader('x-apisports-key', '8ad7209e9e0a016c96f4e199bed14b5c');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    for (let i = 0; i < xhr.response.response[0].league.standings[0].length; i++) {
+      const $newTr = document.createElement('tr');
+      const $newTdBlank = document.createElement('td');
+      const $newTdRank = document.createElement('ins');
+      const $newTdLogo = document.createElement('img');
+      const $newTdTeam = document.createElement('ins');
+      const $newTdWin = document.createElement('td');
+      const $newTdDraw = document.createElement('td');
+      const $newTdLoss = document.createElement('td');
+      const $newTdPoints = document.createElement('td');
+      $teamStandings.appendChild($newTr);
```

Comment:
Once this is broken up into a function, this should be called OUTSIDE the function rather than inside.